testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -537,6 +537,34 @@ protected void checkInformationSchemaViewsForMaterializedView(String schemaName,
                 .containsAll("VALUES '" + viewName + "'");
     }
 
+    @Test
+    public void testMaterializedAndNormalViewsTogether()
+    {
+        if (!hasBehavior(SUPPORTS_CREATE_MATERIALIZED_VIEW) || !hasBehavior(SUPPORTS_CREATE_VIEW)) {
+            throw new SkipException("Either materialized or normal views not supported");
+        }
+
+        String catalogName = getSession().getCatalog().orElseThrow();
+        String schemaName = getSession().getSchema().orElseThrow();
+
+        String normalViewName = "test_views_together_normal_" + randomTableSuffix();
+        assertUpdate("CREATE VIEW " + normalViewName + " AS SELECT * FROM region");
+
+        String materializedViewName = "test_views_together_materialized_" + randomTableSuffix();
+        assertUpdate("CREATE MATERIALIZED VIEW " + materializedViewName + " AS SELECT * FROM nation");
+
+        // both should be accessible via information_schema.views
+        // TODO: actually it is not the cased now hence overridable `checkInformationSchemaViewsForMaterializedView`
+        assertThat(query("SELECT table_name FROM information_schema.views WHERE table_schema = '" + schemaName + "'"))
+                .skippingTypesCheck()
+                .containsAll("VALUES '" + normalViewName + "'");
+        checkInformationSchemaViewsForMaterializedView(schemaName, materializedViewName);
```

Comment:
put this new test between testMaterializedView and checkInformationSchemaViewsForMaterializedView, since both tests use checkInformationSchemaViewsForMaterializedView as a helper now